assignments/array-methods.js

```diff
@@ -55,29 +56,128 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+
+// let fullName = [];
+// for (let i = 0; i < runners.length; i++) {
+//     fullName.push(runners[i].first_name + " " + runners[i].last_name)
+// }
+// console.log(fullName)
+
+
+let fullName2 = []
+runners.forEach(runner =>  fullName2.push(`${runner.first_name} ${runner.last_name}`))
+console.log(fullName2);
```

Comment:
Great job using Template string literals here to render the name information from the runner object. This could also be done using string concatenation as well.